app/models/project/Project.scala

```diff
@@ -100,6 +100,20 @@ class ProjectDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
       parsed <- Fox.combined(r.toList.map(parse))
     } yield parsed
 
+  def findAllWithTaskType(taskTypeId: String)(implicit ctx: DBAccessContext): Fox[List[Project]] =
+    for{
+      accessQuery <- readAccessQuery
```

Comment:
```suggestion
```
(access query is not used in the complex query, so we don’t need to generate it)